Sources/TSCBasic/PathShims.swift

```diff
@@ -61,6 +61,7 @@ public func makeDirectories(_ path: AbsolutePath) throws {
 
 /// Creates a symbolic link at `path` whose content points to `dest`.  If `relative` is true, the symlink contents will
 /// be a relative path, otherwise it will be absolute.
+@available(*, deprecated, renamed: "localFileSystem.createSymbolicLink")
 public func createSymlink(_ path: AbsolutePath, pointingAt dest: AbsolutePath, relative: Bool = true) throws {
```

Comment:
should we keep `relative`? Or is this not important.

Reply:
The relative form is still important, since otherwise, only absolute-path symlinks could be created.

I'm not sure of the provenance of these freestanding functions and whether this should be deprecated or not.  Based on the `currentWorkingDirectory` property below it, it seems that the intent is to deprecate them and replace use with `localFileSystem`, which makes sense.